versioned_docs/version-2023.1/03_server/03_request-server/03_advanced.md

```diff
@@ -6,10 +6,11 @@ keywords: [server, request server, advanced]
 tags:
   - server
   - request server
-  - advanced
+  - requestreply
 ---
 
-
+## Audit tables
+If you want to create a `requestReply`codeblock for an Audit table, you must specify at least one field that is an index field in the underlying table.
```

Comment:
I would add a bit more information here. "If you want to create a `requestReply` codeblock for an Audit table, and you also want to specify a specific set of `request` fields, you need to make sure the fields belong to an index in the underlying audit table.